cloudify_cli/tests/commands/test_secrets.py

```diff
@@ -90,25 +90,56 @@ def test_secrets_create_mutually_exclusive_arguments(self):
         )
         self.assertIn('mutually exclusive with arguments:', outcome.output)
 
-    def test_secrets_export_password_length(self):
+    def test_secrets_export_invalid_password_length(self):
         self.invoke('cfy secrets export -p 1234567',
                     err_str_segment='ERROR: Password must contain at least 8 '
-                                    'characters',
+                                    'characters.',
                     exception=CloudifyValidationError)
 
-    def test_secrets_export_password_empty(self):
-        self.invoke('cfy secrets export -p ' '',
-                    err_str_segment='2',
-                    exception=SystemExit)
+    def test_secrets_export_valid_password(self):
+        self.client.secrets.export = MagicMock(return_value={})
+        self.invoke('cfy secrets export -p 12345678')
+        call_args = list(self.client.secrets.export.call_args)
+        self.assertIn('_password', call_args[1])
```

Comment:
Why do we need this assertion?

Reply:
I think we need this assertion for the next one right after it that asserts the correct password is passed to the rest client. Otherwise, we can get a test failure but it would be because the test couldn't reach the key in the dictionary and not because the test really failed.